environments/prod/find-court-tribunal-service-gov-uk.yml

```diff
@@ -16,3 +16,6 @@ cname:
   - name: "_7c539238d4da39d0fa4242e862930d02.admin"
     ttl: 300
     record: "CA97F4612709C22E162E35D39F71B6A8.32CBBA599F56C6E44500A1CFFF18F733.d85688fce613669775c5.comodoca.com."
+  - name: "admin"
```

Comment:
needs cdnverify and afdverify records for this to work

Reply:
okay, I'll get @damongreen123 to pick up this branch as part of implementing the shutter

Reply:
Added need to verify name